tracer/test/Datadog.Trace.Tests/Configuration/DynamicConfigurationTests.cs

```diff
@@ -110,28 +110,56 @@ public void EnableTracing()
         [Fact]
         public void SetSamplingRules()
         {
-            var tracerSettings = new TracerSettings();
+            // start with local sampling rules only
+            var localSamplingRulesConfig = new[]
+            {
+                new { sample_rate = 0.5,  service = "Service3", resource = "Resource3", },
+            };
+
+            var localSamplingRulesJson = JsonConvert.SerializeObject(localSamplingRulesConfig);
+            var configValues = new Dictionary<string, string> { { "DD_TRACE_SAMPLING_RULES", localSamplingRulesJson } };
+
+            IConfigurationSource configSource = new DictionaryConfigurationSource(configValues);
+            var tracerSettings = new TracerSettings(configSource);
```

Comment:
Just an FYI, `TracerSettings` has a helper for doing this more concisely:
```suggestion
            var tracerSettings = TracerSettings.Create(new(){ { "DD_TRACE_SAMPLING_RULES", localSamplingRulesJson } });
```

Reply:
Thanks!